packages/p2p/src/components/my-profile/my-profile.scss

```diff
@@ -1,10 +1,21 @@
 .my-profile {
+    display: flex;
```

Comment:
`my-profile` wraps around all the tab components, and its parent is `AutoSizer`. Since `my-profile` initially has no styling and its display is `block` by default since it uses `div` element (this applies to `my-profile__content` class as well), this issue causes the list of blocked users table to have `0` height (documentation of `react-virtualized/autosizer` below), thus the table is not visible. Previously this was fixed by setting a fixed height of `32rem`, but now since the list of blocked users should have a scroll height that fills the remaining space as specified by design team (spoke with yunus regarding this), for desktop, `my-profile` and `my-profile__content` should have a `flex` display to allow the table rows to stretch the height of the containers, thus allowing the height to be also automatically adjusted by the `AutoSizer` component
![Screenshot 2022-08-03 at 1 11 14 PM](https://user-images.githubusercontent.com/103016120/182528898-9b83c4b5-c8e9-4762-aee9-42b3e7dbf7a3.png)

Reply:
Link to docs: https://github.com/bvaughn/react-virtualized/blob/master/docs/usingAutoSizer.md

Reply:
Setting `height: 100%` also does not work, only `display: flex`. I have checked other tabs (Payment Methods, Stats, etc) and compared with build from `master`, both screens remain the same, will also add this checklist for QA as well to compare with design if anything changes